source/c.txt

```diff
@@ -75,6 +76,37 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<http://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB server version 5.0 and C Driver version 1.18. When
+you use the Versioned API feature with an official MongoDB driver, you
+can update your driver or server without worrying about backward
+compatibility issues for any commands covered by the Versioned API. To
+use the Versioned API, use the :atlas:`Atlas connection string
+</driver-connection>` for your cluster or a ``localhost`` connection
+string. You also need to specify the version of the Versioned API:
+
+.. code-block:: c
+
+   // Replace the uri string with your MongoDB deployment's connection string.
+   const char *uri_string = "<connection string>";
```

Comment:
I think this is ok, but let's make every example consistent with how we name the placeholder and the comment describing it.

Reply:
it is already consistent across the three drivers here!

Reply:
Ok.
I'm not sure if this is within the scope of this ticket, but we're presenting a term two different ways in the same sentence. I think the intent may have been to refer to the variable name or placeholder name in directing what should be replaced. The C# example does not use a variable so that was a little confusing. I'll let you decide whether you think this could be clearer and whether to include that as part of this ticket.

Reply:
Would it be more clear if the comment said: 

// Replace <connection string> with your MongoDB deployment's connection string.

Reply:
Sure, and then modified the string to be ``"<connection string>"`` or whatever syntax is necessary for that language.